Sources/JExtractSwift/FFM/FFMSwift2JavaGenerator.swift

```diff
@@ -16,27 +16,87 @@ import JavaTypes
 import SwiftSyntax
 import SwiftSyntaxBuilder
 
+package class FFMSwift2JavaGenerator: Swift2JavaGenerator {
+  let log = Logger(label: "ffm-generator", logLevel: .info)
+
+  let analysis: AnalysisResult
+  let swiftModuleName: String
+  let javaPackage: String
+  let swiftOutputDirectory: String
+  let javaOutputDirectory: String
+  let swiftStdlibTypes: SwiftStandardLibraryTypes
+  let symbolTable: SwiftSymbolTable
+
+  var javaPackagePath: String {
+    javaPackage.replacingOccurrences(of: ".", with: "/")
```

Comment:
(Not in this PR, but we should consistently use `PATH_SEPARATOR`, or more preferably `swift-system`'s `FilePath`)